solutions/remotemonitoring/single-vm/docker-compose.dotnet.yml

```diff
@@ -4,8 +4,8 @@ services:
   reverseproxy:
     image: azureiotpcs/remote-monitoring-nginx:testing
     ports:
-      - "80:80"
-      - "443:443"
+      - "80:8080"
+      - "443:8443"
```

Comment:
for consistency with other places where we had to do something similar, I'd suggest using `10080` and `10443`